src/test/java/io/javalin/TestStaticDirectorySlash.kt

```diff
@@ -14,35 +14,35 @@ import org.junit.Test
 
 class TestStaticDirectorySlash {
 
-    private val normalJavalin = Javalin.create()
-            .enableStaticFiles("public")
+    private val normalJavalin = Javalin.create().servlet { it.addStaticFiles("public") }
 
-    private val nonIgnoringJavalin = Javalin.create()
-            .enableStaticFiles("public")
-            .dontIgnoreTrailingSlashes()
+//    private val nonIgnoringJavalin = Javalin.create().servlet { servlet ->
+//            servlet.addStaticFiles("public")
+//            servlet.ignoreTrailingSlashes = false
+//    }
 
     @Test
     fun `normal javalin ignores static directory slashes`() = TestUtil.test(normalJavalin) { _, http ->
         assertThat(http.getBody("/subpage")).isEqualTo("TEST") // ok, is directory
         assertThat(http.getBody("/subpage/")).isEqualTo("TEST") // ok, is directory
     }
 
-    @Test
-    fun `nonIgnoringJavalin (dontIgnoreTrailingSlashes()) doesn't ignore static file slashes`() = TestUtil.test(nonIgnoringJavalin) { _, http ->
-        assertThat(http.getBody("/subpage")).isEqualTo("Not found") // nope, non ignoring and doesnt have slash
-        assertThat(http.getBody("/subpage/")).isEqualTo("TEST") // ok, has slash
-    }
+//    @Test
```

Comment:
Some more commented out tests in this file

Reply:
I'm not sure if we should remove the trailing slash config option. I haven't seen anyone use it though, so it's probably safe.